.circleci/checkout-repos.sh

```diff
@@ -2,9 +2,10 @@
 
 set -euxo pipefail
 
-# Additionally get circleci container public ip address
+# Additionally get circleci container public ip address and hostname
 CONTAINER_IP_ADDRESS=$(wget -qO- http://checkip.amazonaws.com)
 echo $CONTAINER_IP_ADDRESS
+echo $(hostname -s)
```

Comment:
should we remove debug related statements?

Reply:
Agreed, we have a purpose for this too but right now its under investigation if we can get control of such feature:
https://stackoverflow.com/questions/49813313/track-activity-during-build-tests-execution-in-circleci-container